app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/ContactDetailsConfirmAddressController.scala

```diff
@@ -57,63 +56,82 @@ class ContactDetailsConfirmAddressController @Inject() (
         case Some(journeyId) =>
           for {
             incorporationDetails <- incorpIdConnector.getDetails(journeyId)
+            updatedRegistration <- updateBusinessAddress(
+              incorporationDetails.companyAddress.toPptAddress
+            )
           } yield Ok(
             page(form().fill(
                    ConfirmAddress(request.registration.primaryContactDetails.useRegisteredAddress)
                  ),
-                 incorporationDetails.companyAddress
+                 updatedRegistration.businessRegisteredAddress.get
```

Comment:
NIT. Not keen in the use of "get" but I can see how it will always be the address you passed into updateBusinessAddress

Reply:
as discussed simplified a bit so removed 'get' call